### ISSUE TITLE
Hello

### ISSUE THREAD

**Issue by wangqingl** [Orange1105921991/AMSA-CAFF-Net#1]
Hello, first of all, thank you for your contribution to the field of counting. May I ask if your dataset can be made public

**Comment by Orange1105921991** [comment]
Of course, we are willing to share our dataset for academic research, so feel free to contact me if you need it!

**Comment by ddcpowerone** [comment]
> Hello, first of all, thank you for your contribution to the field of counting. May I ask if your dataset can be made public

Hello, have you received the relevant public research data? If you have, could you share it with me? Thank you very much!
